run_build_image.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -xe
+cd -- `dirname ${BASH_SOURCE[0]}`
+ROOT_DIR="."
+NODE_NAME="SEBAKNODE"
+IMAGE_TAG="sebak:test"
+
+docker build -q -t ${IMAGE_TAG} \
+    --build-arg BUILD_MODE='test' \
+    --build-arg BUILD_PKG='./cmd/sebak' \
+    --build-arg BUILD_ARGS='-tags integration -c -o /go/bin/sebak' \
+    ${ROOT_DIR}
+
+docker rmi $(docker images --filter dangling=true -q --no-trunc) || true
+docker images
```

Comment:
That is highly workflow dependent and should not be here IMO

Reply:
The reason why I added this statement is that
the Dockerfile is based on two `From`. Hence, `builder` image remains as unnamed dangling.
But, I understand what you saying.